### ISSUE TITLE
Show single domain management overview page

### ISSUE THREAD

**Issue by Imran92** [Automattic/wp-calypso#96812]
`Will be unblocked when https://github.com/Automattic/wp-calypso/issues/96673 is done`

Tasks:

1. Show the component we currently have in Calypso's wp-admin -> upgrades -> Domains -> Click a domain (Already done in the POC https://github.com/Automattic/wp-calypso/pull/96459, you can take ideas from there. Can also follow site overview)
2. The above is implemented using the displaying middleware(s) introduced in https://github.com/Automattic/wp-calypso/issues/96673
3. Make necessary changes to completely match the Figma design as the screenshot below
4. Add proper routing that's specific for this page
5. It should work for all types of domains i.e. subdomain connection, connected domain, unattached domain etc.

As part of the layout, the sites dashboard is already visible now. We need to match the design ePfz53j71KKDLWtuSm567A-fi-4948_7391

![Image](https://github.com/user-attachments/assets/4b320c7a-01a2-4724-9b54-76f987e6edd3)
